app/assets/javascripts/components/static/home_page.es6.jsx

```diff
@@ -88,6 +88,52 @@ class HomePage extends React.Component {
                 <a className="home__link" href={RouteConstants.staticPages.involvement}>Read more</a>
               </div>
             </div>
+            <div className="testimonial__line-break">
+            </div>
```

Comment:
nit: can you put this all on one line?
`<div className="testimonial__line-break"></div>`